src/CreateAccount.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import NavBar from './component/NavBar';
+import Register from './component/Register';
```

Comment:
Need to fix the name. JS file name convention should be capital need to change the register.js to Register.js